tsconfig-strict.json

```diff
@@ -84,6 +84,14 @@
       "./node_modules/@types"
     ]
   },
+  "exclude": [
```

Comment:
Why wasn't this needed before?

Reply:
I'm uncertain why it got removed in this pull request: https://github.com/oppia/oppia/pull/15578. However, the tsconfig strict tests are currently not passing, so I copied some of the exclusions from the tsconfig.json file.

https://github.com/oppia/oppia/blob/001e4c464289cefeac36445a7adfc05076316e07/tsconfig.json#L59-L66